src/ztex_bcrypt.c

```diff
@@ -56,7 +56,7 @@ static struct device_bitstream bitstream = {
 	// computing performance estimation (in candidates per interval)
 	// (keys * mask_num_cand)/crypt_all_interval per jtr_device.
 	1,	// set by init()
-	262140,	// Absolute max. keys/crypt_all_interval for all devices.
+	1 * 1024*1024,	// Absolute max. keys/crypt_all_interval for all devices.
```

Comment:
If max per device is 65535 and not 65536, should the maximum for all devices possibly be a multiple of 65535 rather than of 65536? Wouldn't a value as above, when we do bump into it, result in tiny last batches?

Reply:
This update removes maximum per device (was 65535, now can be any value).
The above value (1M keys for bcrypt, 2M keys for descrypt - that's template keys, can contain '#' for on-device mask) is because of limited USB bandwith.
That allows to set a cap on USB traffic per crypt_all so it don't run into timeout.